pkg/handlers/webhandler.go

```diff
@@ -30,12 +30,12 @@ const (
 	OpenShiftVersion          = "OPENSHIFT_VERSION"
 	OpenShiftAPIHost          = "OPENSHIFT_API"
 	InstalledServices         = "INSTALLED_SERVICES"
-	WTLocationsDefault        = "https://github.com/integr8ly/tutorial-web-app-walkthroughs#v1.10.1"
+	WTLocationsDefault        = "https://github.com/integr8ly/tutorial-web-app-walkthroughs#v1.12.0"
```

Comment:
@pb82 do we also need to change it in the template? https://github.com/integr8ly/tutorial-web-app-operator/blob/master/deploy/template/tutorial-web-app.yml#L46

Reply:
Yep, good catch